frontend/package.json

```
@@ -46,6 +46,8 @@
     "zod": "^3.24.2"
   },
   "devDependencies": {
+    "@types/react": "^18.3.23",
```

Biome (1.9.4)

[error] The key @types/react was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)